exercise_2.py

```diff
@@ -0,0 +1,4 @@
+origin_list = [1, 2, 4, 5, 6, 9, 7]
+for number in range(1, len(origin_list)):
+    if origin_list[number] != origin_list[number - 1] + 1:
+        print(origin_list[number])
```

Comment:
after finding this number should be an operator break.
Also should be an operator else for loop. (in the case when all number consistent)